neofetch

```diff
@@ -120,19 +120,33 @@ getdistro() {
         "Linux" )
             if type -p lsb_release >/dev/null 2>&1; then
                 case "$distro_shorthand" in
-                    "on")  distro="$(lsb_release -sir 2>/dev/null)" ;;
-                    "off") distro="$(lsb_release -sd 2>/dev/null)"
-                           distro="${distro//\"}" ;;
+                    "on")   distro="$(lsb_release -sir 2>/dev/null)" ;;
+                    "off")  distro="$(lsb_release -sd 2>/dev/null)"
+                            distro="${distro//\"}" ;;
+                    "tiny") distro="$(lsb_release -si 2>/dev/null)" ;;
                 esac
 
             elif type -p guix >/dev/null 2>&1; then
                 distro="GuixSD"
 
             elif type -p crux >/dev/null 2>&1; then
                 distro="$(crux)"
+                case "$distro_shorthand" in
+                    "on")   distro="${distro//version}" ;;
+                    "tiny") distro="${distro//version*}" ;;
+                esac
 
             else
-                distro="$(awk -F 'NAME=' '/^NAME=/ {printf $2}' /etc/*ease)"
+                case "$distro_shorthand" in
+                    "off")
+                        distro="$(awk -F'=' '/^PRETTY_NAME=/ {print $2}' /etc/*ease)" ;;
+
+                    "on")
+                        distro="$(awk -F'=' '/^NAME|VERSION_ID=/ {print $2}' /etc/*ease)" ;;
+
+                    "tiny")
+                        distro="$(awk -F'=' '/^NAME=/ {print $2}' /etc/*ease)" ;;
+                esac
                 distro="${distro//\"}"
```

Comment:
I'm sure we can move this (and the one in line 125) to line 157 (after the if) so we can have just one instance of `distro="${distro//\"}"`. I'm yet to test this, though (and Travis must be tired from me constantly force-pushing).